### DISCUSSION TITLE
[4.0] mod_media: edit an article, insert video 2 functions, 1: to insert iframe/youtube/vimeo - 2: play html5 video local

### DISCUSSION THREAD

**Discussion by nmousedk** [joomla/joomla-cms#47192] [2. Cold Features]
From edit an article. upload video.
Select video from the video-icon (does not work now) - after chose the video icon. you get 2 choices.

1 (youtube/vimeo/iframe
2 local uploaded video, chose the uploaded video, insert html5 code,autoplay/yes/no - replay/yes/no and then the same as an image, border, border color etc etc.

I mean the ordinare stock joomla editor.

**Comment by nmousedk** [comment]
Sorry, it shod started with: Future request.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/19855">issues.joomla.org/tracker/joomla-cms/19855</a>.</sub>

**Comment by nmousedk** [comment]
When I say local video, I meant, after uploading a video, I call this video-file - local.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/19855">issues.joomla.org/tracker/joomla-cms/19855</a>.</sub>

**Comment by ghost** [comment]
Changed Title to be clear Issue is about 4.0.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/19855">issues.joomla.org/tracker/joomla-cms/19855</a>.</sub>

**Comment by ghost** [comment]
Categorie "Feature Request" added at Issue Tracker.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/19855">issues.joomla.org/tracker/joomla-cms/19855</a>.</sub>

**Comment by dgrammatiko** [comment]
Partially fixed with: #34634 (only the local files)
